README.md

```diff
@@ -1 +1,12 @@
+[![Build Status](https://travis-ci.org/andela-gike/Checkpoint-1.svg?branch=master)](https://travis-ci.org/andela-gike/Checkpoint-1)
```

Comment:
Your Travis badge is not available because Travis cannot find the specified spec file in the folder it should be in.